notification/app/notification/controllers/Main.scala

```diff
@@ -39,13 +40,15 @@ final class Main @Inject()(
       case \/-(report) =>
         notificationReportRepository.store(report) map {
           case \/-(_) =>
-            Ok("Ok")
+            Logger.info(s"Notification was sent: $push")
```

Comment:
Yes, I think that could be useful for audit. We only keep 7 days of logs anyway.